app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,55 @@
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all.order(:status)
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    task = Task.new(task: params[:task][:task], description: params[:task][:description], due: params[:task][:due], status: false)
+    task.save
+    redirect_to('/tasks')
```

Comment:
Ideally here we would use `redirect_to tasks_path` instead. It should work the same way as this line does, but is more flexible if we for some reason need to change the URL that maps to the `Tasks#index` action.